src/icons/hide.svg

```diff
@@ -1,7 +1,16 @@
-<svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" viewBox="0 0 16 16">
-  <g fill="none" fill-rule="evenodd" transform="translate(0 1)">
-    <path stroke="#0C0C0D" stroke-opacity=".6" stroke-width="2" d="M1,6.95041322 C1,9.20254695 4.73761678,11.9008264 8,11.9008264 C11.2623832,11.9008264 15,9.20254695 15,6.95041322 C15,4.6982795 11.2623832,2 8,2 C4.73761678,2 1,4.6982795 1,6.95041322 Z"/>
-    <path fill="#6D6D6E" d="M14.0160542,0.675736079 C14.3979633,1.05764518 14.3979633,1.67684218 14.0160542,2.05875128 L2.95193261,13.1228729 C2.5700235,13.504782 1.95082651,13.504782 1.56891741,13.1228729 C1.1870083,12.7409638 1.1870083,12.1217668 1.56891741,11.7398577 L12.633039,0.675736079 C13.0149481,0.293826977 13.6341451,0.293826977 14.0160542,0.675736079 Z"/>
-    <path fill="#6D6D6E" d="M9.60800171,4.46955872 C9.1423119,4.17216421 8.59352469,4 8,4 C6.34314575,4 5,5.34314575 5,7 C5,7.5968592 5.1629633,8.14651554 5.46343994,8.61383057 L9.60800171,4.46955872 Z"/>
-  </g>
+<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" width="16" height="16" viewBox="0 0 16 16">
```

Comment:
Could probably run some optimization on these SVGs, but I just copied them out of the Zeplin mock